g11n-ws/vip-manager-i18n/build.gradle

```diff
@@ -142,9 +143,14 @@ processResources {
 bootJar.doFirst {
      if (project.hasProperty('datatype') && project.getProperty('datatype') == 'pgdb') {
        def propPath = "${buildDir}/resources/main/application.properties"
```

Comment:
def propPath = "${buildDir}/resources/main/application.properties"
       println propPath

       new File(propPath).write(updatedContent, 'UTF-8')

There three lines in the sections are same, please move them out.